pkg/schema/sign.go

```diff
@@ -81,7 +75,7 @@ func NewSigner(pubKeyRef blob.Ref, armoredPubKey io.Reader, privateKeySource int
 	case *openpgp.Entity:
 		privateKey = v
 	case string:
-		privateKey, err = jsonsign.EntityFromSecring(keyID, v)
+		privateKey, err = jsonsign.EntityFromSecring(fingerprint[32:], v)
```

Comment:
what's the intention of this `[32:]` here? not obvious.

Reply:
This PR was based on the master branch before  https://github.com/perkeep/perkeep/pull/1363 was merged. Back then, `EntityFromSecring` would not accept a fingerprint, so I took the last 8 from the fingerprint to get the short id.

However now that https://github.com/perkeep/perkeep/pull/1363 is merged we can pass the full fingerprint, which was my master plan from the start ;)